bootstrap.bat

```diff
@@ -1,4 +1,7 @@
-nuget restore
+@echo off
+
+msbuild /t:restore

```

Comment:
This doesn't seem to work for me. I get the same error as [AppVeyor](https://ci.appveyor.com/project/azeier/hearthstone-deck-tracker) when running bootstrap locally on a clean clone.